components/BookingForm.js

```diff
@@ -0,0 +1,84 @@
+import styles from "./BookingForm.module.css";
+import { useState } from "react";
```

Comment:
For consistency over the project and files imports from external packages should be on top (as all your other files) and internal resources as styles etc separated below

Reply:
Fixed